cookiecutter-xblock/{{cookiecutter.repo_name}}/.github/workflows/ci.yml

```diff
@@ -31,21 +31,7 @@ jobs:
     - name: Install Dependencies
       run: pip install -r requirements/ci.txt
 
-    - name: Create Build
-      run: |
-        rm -rf /tmp/myxblock-xblock
-        XBLOCK=$(pwd) && cd /tmp/ && echo -e '\n\n\n\n\n' | cookiecutter $XBLOCK
-        cd /tmp/myxblock-xblock && make help && pip install -e .
-        cd /tmp/myxblock-xblock && make dev.build
-
     - name: Run Tests
       env:
         TOXENV: ${{ matrix.toxenv }}
       run: tox
-
-    - name: Run coverage
```

Comment:
I understand taking out the build, but taking out codecov seems wrong

Reply:
You're right about this but the workflow also needs to pass which in that case won't. So, we can add the codecov in such a way to make sure that the workflow still works.

Reply:
Can you put this bit back? We've filed https://github.com/openedx/edx-cookiecutters/issues/417 as a separate issue to look at why it fails. We should still remove the other one you took out.

Reply:
Sure, i'll do it.